src/components/Homepage_image/Homepage_image.js

```diff
@@ -0,0 +1,28 @@
+import { getCurrentGameMode } from "../gameMenu";
+import { PHOTO_MODE, NAME_MODE, FAMILY_NAME_MODE } from "../API/newQuestion";
+
+export const homePagePoster = () => {
+  const poster = document.createElement("img");
+  poster.setAttribute("src", "../photos/game-of-thrones-poster.png");
+  poster.classList.add("poster");
+  poster.setAttribute("id", "poster");
+  document.getElementById("app").appendChild(poster);
+  return poster;
+};
+
+export const changePosterByMode = () => {
+  const poster = homePagePoster();
+  getCurrentGameMode((mode) => {
```

Comment:
To increase the readability of the code, create a function and pass it here, do not use an anonymous function here